src/utils/self_test/self_test.c

```diff
@@ -11,10 +11,21 @@
 #include <getopt.h>
 #include <string.h>
 #include <math.h>
+#include <ctype.h>
 
 #include "tests_common.h"
+#include "configini.h"
 #include "daos_errno.h"
 
+/* Section names */
+#define DEFAULT_SCALE_NAME			"scale"
+#define DEFAULT_VALUE_NAME			"default_values"
+#define RAW_DATA_EXTENSION			"raw"
+#define RESULT_EXTENSION			"results"
+
+#define MASTER_VALUE_SIZE			64
+#define INVALID_SCALING				-1.0
+#define MAX_NUMBER_KEYS				200
```

Comment:
visually align

Reply:
It is aligned in the code.  Git parsing issue.